pkg/virt-operator/creation/components/webhooks.go

```diff
@@ -119,6 +119,7 @@ func NewVirtAPIMutatingWebhookConfiguration(installNamespace string) *v1beta1.Mu
 	vmPath := VMMutatePath
 	vmiPath := VMIMutatePath
 	migrationPath := MigrationMutatePath
+	failurePolicy := v1beta1.Fail
 
 	return &v1beta1.MutatingWebhookConfiguration{
```

Comment:
Do we want to take the chance and upgrade to v1 which actually defaults this to Fail?